<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>980</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Circle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>圆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>线(Bresenham)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rectangle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>矩形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BezierCurve">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bezier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Translate">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>平移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rotate">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>旋转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clip">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>裁剪直线2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Polygon">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>20</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>多边形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ellipse">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>椭圆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ARC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>圆弧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZoomIn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>放大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZoomOut">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Triangle">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>三角形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BezierControl">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>更改Bezier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetRP">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>参考点设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteRP">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>参考点删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BSplineCurve">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>0</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B样条</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clip2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>裁剪直线1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Fill">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>填充</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Line2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>线(中点)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DashLine">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>虚线</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="radiusARC">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
